pkgs/http/lib/src/base_request.dart

```diff
@@ -165,4 +165,9 @@ abstract class BaseRequest {
 
   @override
   String toString() => '$method $url';
+
+  /// [cache] property for caching the network request.
+  ///
+  /// Defaults to `default`.
+  String? get cache => HttpCacheOptions.defaultType;
```

Comment:
I don't think that this belongs here since it only applies to `BrowserClient`.  `package:cronet_http` forces the user to set the cache policy for the entire `Client`:
https://pub.dev/documentation/cronet_http/latest/cronet_http/CronetEngine-class.html

Also, why is the type nullable?

Reply:
You are correct, I am thinking of using BrowserClient only that conforms to that option too.

Let me make another commit with cache parameter in the Client.